### PR TITLE
Update RichAssPunks 

### PR DESCRIPTION
**Pull request by Jedanselemia**
Please Update Project !, i am not able to add the Tag by myself, tag should be *RichAssPunks*

### CODE REVIEW

**Comment by MaxJewell** [comment]
Please use the new JSON format in the Readme file on the home page.

**Comment by Jedanselemia** [comment]
Hi, i did validate my policy code on :
https://jsonformatter.curiousconcept.com/#,
i don’t know how to proceed with github or coding at all this is quite hard
to anderstand,

Is this the JSON format needed ?

{ "type": "all", "scripts": [ { "keyHash":
"a5dd237fba529b3c7cf7a71f8dcb576a7459f6578f514acc440a80da", "type": "sig"
}, { "type": "before", "slot": 59883307 } ] }

Thanks for help,

**Comment by Jedanselemia** [comment]
Thanks for reply, is this the JSON format needed ?,how can i update it myself ? 

{
  "type": "all",
  "scripts": [
    {
      "keyHash": "a5dd237fba529b3c7cf7a71f8dcb576a7459f6578f514acc440a80da",
      "type": "sig"
    },
    {
      "type": "before",
      "slot": 59883307
    }
  ]
}

File: RichAssPunks [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (Jedanselemia) [inline RichAssPunks:1]: Add [ and ]  at top and bottom

**Pull request closed without merging**